internal/app/daemon/run/run.go

```diff
@@ -13,10 +12,14 @@ import (
 	"time"
 
 	"github.com/imkira/go-observer"
+	"github.com/pkg/errors"
 	"github.com/tidwall/gjson"
 	"github.com/tidwall/sjson"
 	"google.golang.org/grpc"
 
+	"github.com/breml/logstash-config/ast"
+	"github.com/breml/logstash-config/ast/astutil"
```

Comment:
Shouldn't these be in one import group?

Reply:
Do you mean the `github.com/breml` imports should be in the same import group as all the other non LFV 3rd party imports?

I guess from a LFV point of view, this is correct. For my own projects, I have VSCode configured to separate everything with `github.com/breml` prefix, so I need to overwrite this setting for LFV. 😀